v2/pkg/common/annotations/annotations.go

```diff
@@ -3,4 +3,15 @@
 
 package annotations
 
-const PerResourceSecret = "serviceoperator.azure.com/credential-from"
+import "github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+
+const (
+	PerResourceSecret = "serviceoperator.azure.com/credential-from"
+	OwnedByAnnotation = "serviceoperator.azure.com/owned-by"
```

Comment:
I think `owner-name` or `owner` would be clearer than `owned-by`.

Do we also need to think about `owner-group-kind` or something as well, for extension resources such as roleassignment?

Reply:
Not having `owner-group-kind` forces the user to do another query to discover the owner type if they don't already know it. I think there's value in making it easy/obvious.

Reply:
Sure! and that applies to only resources which are KubernetesReferences?